gapis/api/vulkan/extensions/khr_descriptor_update_template.api

```diff
@@ -122,102 +121,110 @@ sub ref!DescriptorUpdateTemplateObject createDescriptorUpdateTemplate(
     const VkDescriptorUpdateTemplateCreateInfo* pCreateInfo,
     VkDescriptorUpdateTemplate*                 pDescriptorUpdateTemplate,
     bool fromKHR) {
-    if !(device in Devices) { vkErrorInvalidDevice(device) }
-    if pCreateInfo == null { vkErrorNullPointer("pDescriptorUpdateTemplate") }
-    info := pCreateInfo[0]
-    if !(info.descriptorSetLayout in DescriptorSetLayouts) { vkErrorInvalidDescriptorSetLayout(info.descriptorSetLayout) }
-    obj := new!DescriptorUpdateTemplateObject(
-        Device: device,
-        Flags: info.flags,
-        TemplateType: info.templateType,
-        DescriptorSetLayout: info.descriptorSetLayout,
-        PipelineBindPoint: info.pipelineBindPoint,
-        PipelineLayout: info.pipelineLayout,
-        SetNumber: info.set,
-        FromKHR: fromKHR,
-    )
-
-    obj.Entries = rewriteDescriptorUpdateEntries(
-        info.descriptorUpdateEntryCount,
-        info.pDescriptorUpdateEntries,
-        info.descriptorSetLayout)
-
+    obj := new!DescriptorUpdateTemplateObject()
+    if !(device in Devices) {
```

Comment:
I would switch the device section back, so it matches every other command.

Reply:
Done